safe-modules/liveness-module-v2.md

```diff
@@ -0,0 +1,139 @@
+threshold, the challenge fails and the Safe receives the bond as a reward.
+
+The Account Liveness Challenge addresses both of the ways in which a Safe can fail to be live. If
+sufficiently many honest users cannot sign that the account cannot reach a threshold, the account
+falls back to the fallback owner. If sufficiently many users are malicious that the account cannot
+reach a threshold, the honest users can refuse to sign and force a fallback.
+
+## Alternatives Considered
+
+<!-- Describe any alternatives that were considered during the development of this design. Explain
+why the alternative designs were ultimately not chosen and where they failed to meet the product
+requirements. -->
+
+### Owner Liveness Challenge
+
+We previously considered a model that would allow users to challenge either the entire Safe (as
+happens in the Account Liveness Challenge) or an individual owner (i.e., an Owner Liveness
+Challenge). An Owner Liveness Challenge would be more similar to the behavior of the existing
+`LivenessModule`, where a user could target a specific owner and require that owner respond or be
+removed from the Safe.
+
+The Owner Liveness Challenge faces a number of problems:
+
+- It creates a second challenge path.
+- It only solves the honest user case, whereas the Account Liveness Challenge solves both cases.
+- It requires that the Safe configure various new parameters like minimum owner count and threshold
+  percentage which creates more surface area for potential issues.
+
+Additionally, in most cases, the Owner Liveness Challenge would be unused because the Safe would
+prefer to simply remove an owner that is unable to sign.
+
+## Risks and Uncertainties
+
+<!-- Explain any risks and uncertainties that this design includes. Highlight aspects of the design
+that remain unclear and any potential issues we may face down the line. -->
+
+### Fallback Ownership
+
+Both the existing `LivenessModule` and this updated iteration rely on fallback ownership. In the
+case that the challenge fails, ownership of the account must be transferred to some fallback
+address. We believe this is acceptable and reasonable. Most multisig accounts can simply name the
+Optimism Foundation multisig as a fallback, and the Optimism Foundation multisig can name the
+Security Council as a fallback.
+
+One open question is what the Security Council should set as a fallback. We'd likely want this to
+be another multisig operated by an independent and well-trusted entity.
+
+### Blunt Instrument
+
+It should be noted that the updated `LivenessModule` is more of a blunt instrument than the
+existing module. We no longer have the ability to remove specific owners, we instead trigger a
+fallback immediately if a threshold cannot be reached. This is less surgical, but we believe it is
+equally effective and ultimately safer/simpler than the existing module because users don't need to
+account for as many parameters.
```

Comment:
You might want to note that if at some point removing individual inactive signers becomes too much of a burden for the active signers, it would be possible to implement a module that challenges an individual signer to show liveness or be removed. Or you can also add it to the out of scope section.

Reply:
This is something we wanted to avoid (this would be the Owner Liveness Challenge). I don't think it would be too burdensome, ideally we shouldn't need to develop this. A single transaction to swap an owner shouldn't be too much effort for signers.

Reply:
I'm a fan of the blunt approach. It's much easy to understand, and easier to not run into unanticipated situations in the real world across many safes.